README.md

```diff
@@ -1,24 +1,26 @@
 # Classnames
 
-[![NPM version](https://badgen.net/npm/v/classnames)](https://www.npmjs.com/package/classnames)
-![Node.js CI](https://github.com/JedWatson/classnames/workflows/Node.js%20CI/badge.svg)
-[![NPM Weekly Downloads](https://badgen.net/npm/dw/classnames)](https://www.npmjs.com/package/classnames)
-[![License](https://badgen.net/npm/license/classnames)](https://www.npmjs.com/package/classnames)
-[![Supported by Thinkmill](https://thinkmill.github.io/badge/heart.svg)](https://thinkmill.com.au/?utm_source=github&utm_medium=badge&utm_campaign=classnames)
-
-A simple JavaScript utility for conditionally joining classNames together.
-
-Install with [npm](https://www.npmjs.com/), [Bower](https://bower.io/), or [Yarn](https://yarnpkg.com/):
-
+> A simple JavaScript utility for conditionally joining classNames together.
+
+<p>
+  <a aria-label="NPM version" href="https://www.npmjs.com/package/@keystone-6/core">
```

Comment:
These links are fixed in  a follow-up pull request, https://github.com/JedWatson/classnames/pull/305